play-scala-isolated-slick-example/app/controllers/UserController.scala

```diff
@@ -0,0 +1,140 @@
+package controllers
+
+import com.example.user.{User, UserDAO}
+import models.UserRequest
+import play.api.i18n.{Lang, Messages}
+import play.api.libs.json.Json
+import play.api.mvc._
+
+import java.time.Instant
+import java.util.UUID
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class UserController @Inject() (userDAO: UserDAO, cc: ControllerComponents)(
+    implicit ec: ExecutionContext
+) extends AbstractController(cc) {
+  /**
+   * An implicit for default Languages for the Inputs to default to english.
+   */
+  implicit val defaultLanguage: Messages = cc.messagesApi.preferred(Seq(Lang("en")))
+
+  /**
+   * GET - Get the List of all the users from the userDao
+   * @return Html View of Index with all the users
+   */
+  def index: Action[AnyContent] = Action.async { implicit unused =>
```

Comment:
I think it's better to actually name variables what they represent, even if they are unused. Usually you see in the IDE anyway that they are not used.
Can you rename all? Thanks.
```suggestion
  def index: Action[AnyContent] = Action.async { implicit request =>
```